ext/Logger.php

```diff
@@ -63,26 +65,52 @@ public function _initialize()
         if (!class_exists('\Monolog\Logger')) {
             throw new ConfigurationException("Logger extension requires Monolog library to be installed");
         }
-        $this->path = $this->getLogDir();
+    }
+
+    public static function getLogger()
+    {
+        if (self::$logger) {
+            $logger = self::$logger;
+        } else {
+            $logger = self::getDefaultLogger();
```

Comment:
Logger should be created only on codeception events. Before suite or before init, whatever.